packages/stylelint-plugin-meteor/src/rules/prefer-border-token/prefer-border-token.test.ts

```diff
@@ -86,5 +90,14 @@ testRule({
       endLine: 1,
       endColumn: 24,
     },
+    {
+      code: ".a { border-bottom-right-radius: 1px calc(2px / 0.18); }",
+      message:
+        'Unexpected hard-coded border radius of "1px" (meteor/prefer-border-token)',
+      line: 1,
+      column: 6,
+      endLine: 1,
+      endColumn: 55,
+    },
```

Comment:
Looks good, only think I would change is to rename `border-bottom-right-radius` to border-radius as the bottom right value can only accept one value.